scripts/generate-conda-recipe.py

```diff
@@ -0,0 +1,34 @@
+import os
+import shutil
+import sys
+
+root_path = 'conda'
+package_name = 'jina-hubble-sdk'
+tag = None
+
+def generate_meta_yaml_from_pypi(package_name: str, tag: str or None):
+    """Generate conda meta yaml from pypi"""
+
+    maintainers = 'mapleeit nomagick delgermurun Andrei997 floralatin'
```

Comment:
Is it necessary to have this info twice both here and in `meta.yml`?

Reply:
This info  `maintainers = 'mapleeit nomagick delgermurun Andrei997 floralatin` in meta.yml,
 It is our configuration and the result of running `python generate-conda-recipe`.